### PR TITLE
added issue template

### PR DESCRIPTION
**Pull request by harshil1712**
I've added an issue template that I feel we should follow throughout the development. Please have a look and let me know if we need any changes.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+
+## Detailed Description
+<!--- Provide a detailed description of the change or addition you are proposing -->
+
+## Context
+<!--- Why is this change important to you? How would you use it? -->
+<!--- How can it benefit other users? -->
+
```
Comment (ayushjainrksh) [inline .github/ISSUE_TEMPLATE/issue_template.md:15]: A section for adding screenshots `Screenshots of the issue`

Comment (harshil1712) [reply .github/ISSUE_TEMPLATE/issue_template.md:15]: Yup we can do that. Great idea. Let me quickly add that.

Comment (harshil1712) [reply .github/ISSUE_TEMPLATE/issue_template.md:15]: Now that I think of it, I think it is more suitable for PR template. What do you think?

Comment (ayushjainrksh) [reply .github/ISSUE_TEMPLATE/issue_template.md:15]: Well, I've seen screenshots on both issue and PR templates. So it might be a good idea to keep it.

**Pull request merged**